python/api/api.py

```diff
@@ -58,26 +58,42 @@ def main():
                 'timeseries': {
                     'type': 'local'
                 }
-            }
+            },
+            'nodes_with_access': [
+            ]
         }
 
     print('~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~')
     print(json.dumps(labbox_config, indent=4))
     print('~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~')
 
-    local_job_handlers = dict(
-        default=hi.ParallelJobHandler(4),
-        partition1=hi.ParallelJobHandler(4),
-        partition2=hi.ParallelJobHandler(4),
-        partition3=hi.ParallelJobHandler(4),
-        timeseries=hi.ParallelJobHandler(4)
-    )
+    nodes_with_access = labbox_config.get('nodes_with_access', [])
```

Comment:
the labbox config now has a nodes_with_access field where we can specify which kachery nodes have permission to write to the default workspace on the labbox node. This allows us to import recordings/sortings into the cloud workspace from a workstation.